bonsai/sdk/src/alpha.rs

```diff
@@ -49,6 +49,7 @@ pub enum SdkErr {
 
 /// Collection of serialization object for the REST api
 pub mod responses {
+    use risc0_groth16::Seal;
```

Comment:
I saw that you exported this here: https://github.com/risc0/risc0/pull/1420/files#diff-17e9437e9950f794a22d8fcae4e2e99e315c137e1eb2950612bf0ea774188f4dR136 so IIUC, you can use it here by `use risc0-zkvm::Groth16Seal as Seal` and `risc0-groth16` from this crate's Cargo.toml

Reply:
I avoided doing that as risc0-zkvm depends on bonsai-sdk, so I've imported the risc0-groth16 crate to brake the circular dependency